patches/server/0164-Fix-MC-117075-TE-Unload-Lag-Spike.patch

```diff
@@ -1,42 +1,158 @@
 From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
-From: mezz <tehgeek@gmail.com>
-Date: Wed, 9 Aug 2017 17:51:22 -0500
+From: MrPowerGamerBR <git@mrpowergamerbr.com>
+Date: Sun, 26 Nov 2023 13:02:16 -0300
 Subject: [PATCH] Fix MC-117075: TE Unload Lag Spike
```

Comment:
Not related to the changes, but the patch should be renamed to say block entities instead of TE/tile entities